pom.xml

```diff
@@ -431,6 +458,34 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.openapitools</groupId>
+                <artifactId>openapi-generator-maven-plugin</artifactId>
+                <version>${openapitools-version}</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <generatorName>spring</generatorName>
+                            <inputSpec>${project.basedir}/src/test/resources/contracts/e-bank/openapi.yml</inputSpec>
+                            <configOptions>
+                                <sourceFolder>src/main/java</sourceFolder>
+                                <library>spring-boot</library>
+                                <dateLibrary>java8</dateLibrary>
+                                <java8>true</java8>
```

Comment:
Shouldn't we use Java 11?

Reply:
Actually I didn't notice in documentation java11, only java8, but I can check it